frontend/i18n/pl/home.js

```diff
@@ -0,0 +1,18 @@
+export default {
+  mainTitle: 'Adnotacja tekstowa dla ludzi',
+  getStarted: 'Rozpocznij',
+  startAnnotation: 'Rozpocznij adnotację',
+  featuresTitle: 'Najlepsze funkcje',
+  featuresTitle1: 'Współpraca zespołowa',
+  featuresText1: 'Adnotacja z kolegami z drużyny',
+  featuresTitle2: 'Dowolny język',
```

Comment:
It is better to write it `Anotacja z kolegami z zespołu` because zespół can be used for team in the sense of work/office, while drużyny is more for sports I think 😅